1.9/usage/service-discovery/dns-overview.md

```diff
@@ -0,0 +1,68 @@
+---
+post_title: DNS Quick Reference
+nav_title: DNS Reference
+menu_order: 20
+---
+
+We've got a lot going on when it comes to DNS and this article aims to
+summarize all that information in a single place.
+
+# Example Setup
+
+To make things more concrete, we'll use this example application:
+* (Group) `outergroup` > (Group) `innergroup` > (Service Name) `myapp`
```

Comment:
Well this one is a reference sheet rather than a full tutorial, and the rest of the JSON would just be a distraction. I also want to keep this as short and to the point as possible.